doc/Configuration.md

```diff
@@ -22,12 +22,13 @@
 
```

Comment:
Mention of `.buildrc` at line 5, but no mention of its location. Perhaps this is better place to mention default and alternative locations rather than in '## `.buildrc` variables'

Reply:
Since I consider the use of BUILDER_CONFIG a very rare edge case, it doesn't make sense to mention it everywhere and even before the user has a chance to understand Builder itself. I added the `~/` back in since it seems confusing to you. Originally I removed it in this PR to be more open as to where the .buildrc actually is. Main description of the feature is under Installation -> Multi-configuration Environments, since sections there describe various special edge cases.